insights/combiners/sudoers.py

```diff
@@ -41,8 +41,10 @@ class Sudoers(SudoersBase):
     """
     def __init__(self, sudoers):
         self.lines = []
+        self.data = {}
```

Comment:
@TZ3070 , we better also add this new attribute in the docstrings' `Attributes` part, just as `lines(list)` .

Reply:
@JoySnow Updated